MagneticField/include/ActsLUXEPipeline/LUXEMagneticField.hpp

```diff
@@ -12,6 +12,10 @@
 
 namespace LUXEMagneticField {
 
+struct GridOptions {
+    std::vector<unsigned int> bins;
+    std::vector<std::pair<float,float>> limits;
+};
 /// 3D equidistant binning
 using Grid_t =
         Acts::Grid<Acts::Vector3, Acts::detail::EquidistantAxis
```

Comment:
Should also we add `using` for the `variableGrid`, i.e. the one that takes the `VariableAxis`?